.travis.yml

```diff
@@ -35,10 +33,11 @@ before_install:
   - composer self-update
   - if [ "$DEPS" = "dev" ]; then perl -pi -e 's/^}$/,"minimum-stability":"dev"}/' composer.json; fi
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require symfony/symfony:${SYMFONY_VERSION} --no-update; fi
+  - phpunit --version
 
 install: composer update --prefer-dist $COMPOSER_FLAGS
 
-script: vendor/bin/simple-phpunit
```

Comment:
Please revert this change. We use `simple-phpunit` to avoid building PHPunit with Yaml.

You can change the version that's used by simple-phpunit by setting the `SYMFONY_PHPUNIT_VERSION` env var. Please set that one to 5.6 (or 5.7) instead.